api/src/main/java/org/pmiops/workbench/elasticsearch/ElasticDocument.java

```diff
@@ -59,11 +59,11 @@ String lower() {
   public static final Map<String, Object> PERSON_SCHEMA = ImmutableMap.<String, Object>builder()
       .put("birth_datetime", esType(ElasticType.DATE))
       .put("gender_concept_id", esType(ElasticType.KEYWORD))
-      .put("gender_concept_name", esType(ElasticType.TEXT))
+      .put("gender_concept_name", esType(ElasticType.KEYWORD))
```

Comment:
Note: I was wrong about the keyword subfield - that's just the default ES behavior. We could add a field type like this, but I don't think it's worth indexing this as text anyways (it's a small handful of values).

```
                "type" : "text",
                "fields" : {
                  "keyword" : {
                    "type" : "keyword",
                    "ignore_above" : 256
                  }
                }
```